packages/Header/CHANGELOG.md

```diff
@@ -0,0 +1,13 @@
+# Changelog
+
+## Unreleased
+
+#### Breaking Changes
+#### Improvements
+#### Bug fixes
+
```

Comment:
To remove, you already define it just below

Reply:
Fixed